frontend/src/components/learner/browser/CourseCard.tsx

```diff
@@ -42,11 +42,11 @@ const CourseCard = ({
     ) || 0;
   const expanded = focused || hovered;
 
-  const StatusTag = () => {
-    if (!status) {
+  const StatusTag = (courseProgress: string | null) => {
```

Comment:
I'm fine with this not being a component (though ideally it would be) but it would definitely be nice to extract this out of line to the top level, or even to a separate file.